babel.config.js

```diff
@@ -26,7 +27,11 @@ module.exports = api => {
       ],
       '@babel/preset-react',
     ],
-    plugins: ['@babel/plugin-proposal-class-properties', 'babel-plugin-lodash'],
+    plugins: clean([
+      '@babel/plugin-proposal-class-properties',
+      'babel-plugin-lodash',
+      !isStorybook && 'babel-plugin-transform-react-remove-prop-types',
```

Comment:
We've been relying on PropTypes for development errors.

Example:

```jsx
<RefinementList />

// Warning: Failed prop type: The prop `attribute` is marked as required in 
// `AlgoliaRefinementList(RefinementListWidget)`, but its value is `undefined`.
```

We will lose this information with this config.

Can we benefit from this change only in the production bundle?

Reply:
Yes, my intent was only to touch the UMD bundles but seems the CJS and ES builds are also affected.

I will update my change to only target the UMD bundles